src/drawer.multicanvas.js

```diff
@@ -456,9 +456,10 @@ export default class MultiCanvas extends Drawer {
      * rendered
      * @param {function} fn The render function to call, e.g. `drawWave`
      * @param {number} drawIndex The index of the current channel after filtering.
+     * @param {number?} normalizedMax maximum modulation value accross channels for use with relativeNormalization. Ignored when undefined
      * @returns {void}
      */
-    prepareDraw(peaks, channelIndex, start, end, fn, drawIndex) {
+    prepareDraw(peaks, channelIndex, start, end, fn, drawIndex, normalizedMax) {
```

Comment:
normalizedMax param should be documented.

Reply:
I added  ESDocs documentation  for the parameter on line 459 in the original pull request. Is there another place I should document the parameter, or is the documentation insufficient?

Current docs:  `* @param {number?} normalizedMax Maximum modulation value across channels for use with relativeNormalization. Ignored when undefined`

Reply:
ah yes.